modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/LayoutProvider.es.js

```diff
@@ -375,16 +373,7 @@ class LayoutProvider extends Component {
 	}
 
 	_handleFieldHovered(fieldHovered) {
-		const {fieldName} = fieldHovered;
-
-		if (fieldName) {
-			this.setState(
-				handleFieldHovered(this.props, this.state, fieldName)
-			);
```

Comment:
@alinedoleron, why is it no longer needed? Can you please give me more details. Thanks.

Reply:
@natocesarrego, I've added this code in the PR before, just in case app builder should pass something different on this method. But after talking to the team and debugged on app builder form I noticed it works the same way of forms.